fpga/Makefile

```diff
@@ -90,11 +90,14 @@ fpga_common_script_dir := $(fpga_dir)/common/tcl
 #########################################################################################
 # setup misc. sim files
 #########################################################################################
+SIM_FILE_REQS += \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/EICG_wrapper.v
+
 # copy files but ignore *.h files in *.f (match vcs)
-$(sim_files): $(SIM_FILE_REQS) | $(GEN_COLLATERAL_DIR)
-	cp -f $^ $(GEN_COLLATERAL_DIR)
+$(sim_files): $(SIM_FILE_REQS) $(ALL_MODS_FILELIST) | $(GEN_COLLATERAL_DIR)
+	cp -f $(SIM_FILE_REQS) $(GEN_COLLATERAL_DIR)
 	$(foreach file,\
```

Comment:
I'm happy to merge this now, but I think this is only a temporary fix. The rules should still work if SIM_FILE_REQS is an empty list.

Reply:
Agreed, I kind of came up with the easiest fix that will work quickly

Reply:
If this is blocking, let's merge this now. I found this issue as well and put this on my TODO list but I won't be able to get to this till afternoon tomorrow.